phpBB/phpbb/template/asset.php

```diff
@@ -152,6 +152,13 @@ public function get_path()
 	*/
 	public function set_path($path, $urlencode = false)
 	{
+		// Since 1.7.0 Twig returns the real path of the file. We need it to be relative to the working directory.
+		$real_root_path = realpath('.') . DIRECTORY_SEPARATOR;
+		if ($real_root_path && substr($path . DIRECTORY_SEPARATOR, 0, strlen($real_root_path)) === $real_root_path)
```

Comment:
Since `$real_root_path` contains at least `DIRECTORY_SEPARATOR`, the `$real_root_path && ` part is unnecessary